devices.js

```diff
@@ -10072,6 +10082,57 @@ const devices = [
             await bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
         },
     },
+    {
+        zigbeeModel: ['eTRV0100'],
+        model: 'eTRV0100',
+        vendor: 'Danfoss',
+        description: 'Danfoss Ally Thermostat',
+        supports: 'temperature, heating system control',
+        fromZigbee: [
+            fz.battery,
+            fz.thermostat_att_report,
+            fz.danfoss_thermostat_att_report,
+        ],
+        toZigbee: [
+            tz.thermostat_occupied_heating_setpoint,
+            tz.thermostat_local_temperature,
+            tz.danfoss_mounted_mode,
+            tz.danfoss_thermostat_orientation, tz.danfoss_algorithm_scale_factor,
+            tz.danfoss_heat_available, tz.danfoss_heat_request,
+            tz.danfoss_day_of_week, tz.danfoss_trigger_time,
+            tz.danfoss_window_open,
+            tz.danfoss_display_orientation,
+            tz.thermostat_keypad_lockout,
```

Comment:
Does this thermostat also supports system_mode? (asking because all other thermostats do). Try adding `tz.thermostat_system_mode` and send `{"system_mode": "off"}`, `{"system_mode": "auto"}`,`{"system_mode": "heat"}`

Reply:
I was not able to verify that it does. I could not get it to change to manual or off mode in any reproducible manner

A.